app/assets/javascripts/scp-related-genes-ideogram.js

```diff
@@ -72,6 +85,31 @@ function showRelatedGenesIdeogram() { // eslint-disable-line
   ideoContainer.classList = 'show-related-genes-ideogram'
 }
 
+/**
+ * Get summary of related-genes ideogram that was just loaded or clicked
+ */
+function getRelatedGenesAnalytics(ideo) {
+  const props = Object.assign({}, ideo.relatedGenesAnalytics)
+
+  // Use DSP-conventional name
+  props['perfTime'] = props.time
+  delete props['time']
+
+  props['species'] = window.SCP.taxon
+  return Object.assign({}, props)
+}
+
+/**
+ * Callback to report analytics to Mixpanel.
+ * Helps profile denominator of click-through-rate
+ */
+function onPlotRelatedGenes() {
+  const ideo = this // eslint-disable-line
```

Comment:
seems like you should either use a more descriptive name than 'ideo' (and add a comment about what it is) or just drop this line and use 'this' on the line below.